spark-kinesis-dynamodb/src/main/scala/org/apache/spark/streaming/kinesis/KinesisCheckpointer.scala

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
thnx, this commit makes much sense, the next one then just shows diffs for our purpose, I thought about just overloading the classes over upstream, but rename and patch sounds better

Reply:
Overriding the classes would have been a great approach too, but I don’t think it was applicable here because many of the internal things are `private` and can not be accessed from subclasses.